applications/GeoMechanicsApplication/tests/cpp_tests/custom_processes/test_fix_water_pressures_above_phreatic_line.cpp

```diff
@@ -193,8 +192,8 @@ KRATOS_TEST_CASE_IN_SUITE(TestFixWaterPressureAbovePhreaticLine_FreesNodesWhenTh
     EXPECT_FALSE(r_model_part.GetNode(2).IsFixed(WATER_PRESSURE));
 
     // Act
-    // Move node 1 below phreatic line using total displacements (without moving the mesh, since move_mesh is false)
-    r_model_part.GetNode(1).FastGetSolutionStepValue(TOTAL_DISPLACEMENT)[1] = -1.0;
+    // Move node 1 below phreatic line
+    r_model_part.GetNode(1).Coordinates()[1] = -1.0;
```

Comment:
```suggestion
    r_model_part.GetNode(1).Y() = -1.0;
```
Easier to read for me, because it avoids thinking about indices of directions.

Reply:
I completely agree. Changed that.